### PR TITLE
a few changes in the FEM example

### PR DESCRIPTION
**Pull request by jorgepz**
## PR description
a few changes in the FEM example

### CODE REVIEW

**Comment by lucaferranti** [comment]
shouldn't I merge the other PR then?

**Comment by jorgepz** [comment]
I will do a few commits and then set it ready for review/merge

**Comment by lucaferranti** [comment]
A-haa, I didn't see you opened the PR to the other branch, then it's ok, great!

File: docs/literate/applications/FEM_example.jl [hunk 1]
```diff
@@ -1,19 +1,20 @@
-# # A simple FEM application
-# The Finite Element Method is widely used to solve PDEs in Engineering applications and particularly in Structural Analysis problems [[BAT14]](@ref). The procedure consists discretizing the domain into _elements_ and assembly a system of balance equations. For linear problems, this system can be usually written as
+# # Applications of ILA to FEM analysis
```
Comment (lucaferranti) [inline docs/literate/applications/FEM_example.jl:1]: ILA is our internal jargon, maybe something like "Application of interval linear algebra to FEM analysis"? or "Parametric interval linear systems in FEM analysis"

**Review by lucaferranti — APPROVED** [review]
I left a small comment, otherwise LGTM. I'll also reactivate the normal tests here so it can be merged in the other PR

File: docs/literate/applications/FEM_example.jl (hunk 1 above)
Comment (jorgepz) [reply docs/literate/applications/FEM_example.jl]: done!

**Pull request merged**